src/AWS/Signers/V4.elm

```diff
@@ -52,7 +54,7 @@ formatDate date =
     date
         |> formatISO8601
         |> Regex.replace All
-            (regex "([-:\\]|\\.\\d{3})")
+            (regex "[-:]")
```

Comment:
Tried this out in the `elm repl` and it works:

```elm
replace All (regex "([-:]|\\.\\d{3})") (\_ -> "") "2017-02-26T18:16:38Z"
```

produces `20170226T181638Z`